.github/workflows/check_pr_title.yml

```diff
@@ -32,7 +34,7 @@ jobs:
     timeout-minutes: 5
 
     steps:
-      - uses: morrisoncole/pr-lint-action@51f3cfabaf5d46f94e54524214e45685f0401b2a
+      - uses: morrisoncole/pr-lint-action@v1.7.1
```

Comment:
Pinning to specific commits is intentional and good practice for security, as tags can change but commits cannot.

Reply:
Internally, we pin everything, we should do the same here as well.